Add Style related documentation on interactive message buttons (#871)
mattermost/mattermost-developer-documentation: site/content/integrate/admin-guide/admin-interactive-messages/_index.md

![image](interactive_message.gif)

> Button actions support a style parameter to change the color of the button. The possible values for style are: `good`, `warning`, `danger`, `default`, `primary` and `success`. It's also possible to pass a theme variable or a hex color, but we discourage this approach because it won't be resilient against theme changes.

Reviewer: ```suggestion
Button actions support a style parameter to change the color of the button. The possible values for style are: `good`, `warning`, `danger`, `default`, `primary`, and `success`. It's also possible to pass a theme variable or a hex color, but we discourage this approach because it won't be resilient against theme changes.
```

> Button actions support a style parameter to change the color of the button. The possible values for style are: `good`, `warning`, `danger`, `default`, `primary` and `success`. It is possible to pass also a theme variable or a hex color, but we discourage this option because it will be not resilient against theme changes.

Reviewer 2: ```suggestion
Button actions support a style parameter to change the color of the button. The possible values for style are: `good`, `warning`, `danger`, `default`, `primary` and `success`. It's also possible to pass a theme variable or a hex color, but we discourage this approach because it won't be resilient against theme changes.
```

![image](interactive_button_style.png)

> The actions used for the previous example are the following:

Reviewer 2: ```suggestion
The actions used in the previous example include the following:
```

```
[
  {
    "id": "vote0",
    "type": "button",
    "name": "Yes",
    "style": "default"
  },
  {
    "id": "vote1",
    "type": "button",
    "name": "No",
    "style": "primary"
  },
  {
    "id": "addOption",
    "type": "button",
    "name": "Add Option",
    "style": "warning"
  },
  {
    "id": "deletePoll",
    "type": "button",
    "name": "Delete Poll",
    "style": "success"
  },
  {
    "id": "endPoll",
    "type": "button",
    "name": "End Poll",
    "style": "danger"
  }
]
```

## Message Menus

> Similar to buttons, add message menus as `actions` in your integration [message attachments](https://docs.mattermost.com/developer/message-attachments.html).

Reviewer: ```suggestion
Similar to buttons, add message menus as `actions` in your integration [message attachments]({{< ref "message-attachments" >}}).
```

![image](interactive_message.gif)

Button actions support a style parameter to change the color of the button. The possible values for style are: `good`, `warning`, `danger`, `default`, `primary`, and `success`. It's also possible to pass a theme variable or a hex color, but we discourage this approach because it won't be resilient against theme changes.

![image](interactive_button_style.png)

The actions used in the previous example include the following:

```
[
  {
    "id": "vote0",
    "type": "button",
    "name": "Yes",
    "style": "default"
  },
  {
    "id": "vote1",
    "type": "button",
    "name": "No",
    "style": "primary"
  },
  {
    "id": "addOption",
    "type": "button",
    "name": "Add Option",
    "style": "warning"
  },
  {
    "id": "deletePoll",
    "type": "button",
    "name": "Delete Poll",
    "style": "success"
  },
  {
    "id": "endPoll",
    "type": "button",
    "name": "End Poll",
    "style": "danger"
  }
]
```

## Message Menus

> Similar to buttons, add message menus as `actions` in your integration [message attachments]({{< ref "message-attachments" >}}).

Reviewer: ```suggestion
Similar to buttons, add message menus as `actions` in your integration [message attachments]({{< ref "admin-message-attachments" >}}).
```
